security/v1/types.go

```diff
@@ -24,6 +24,7 @@ type SecurityContextConstraints struct {
 	metav1.TypeMeta `json:",inline"`
 	// Standard object's metadata.
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	// +nullable
```

Comment:
I don't think we want `nullable` on `ObjectMeta`.

Reply:
right, fixed.